php/savesettings.php

```diff
@@ -48,6 +48,37 @@ function validDomain($domain_name)
 			"8.20.247.20" => "Comodo"
 		];
 
+$adlistsdefault = [];
+$adlistsuser = [];
+function readAdlists($listname)
+{
+	// Reset list
+	$list = [];
+	$handle = @fopen("/etc/pihole/adlists.".$listname, "r");
+	if ($handle)
+	{
+		while (($line = fgets($handle)) !== false)
+		{
+			if(substr($line, 0, 5) === "#http")
+			{
+				// Commented list
+				array_push($list, [false,rtrim(substr($line, 1))]);
+			}
+			elseif(substr($line, 0, 4) === "http")
+			{
+				// Active list
+				array_push($list, [true,rtrim($line)]);
+			}
+		}
+		fclose($handle);
+	}
+	return $list;
+}
+
+	// Read available adlists
+	$adlistsdefault = readAdlists("default");
```

Comment:
Shouldn't this be reading `list` instead of `default`? The web interface is modifying default instead of list.